app/views/services/govbox/register_step1.html.erb

```diff
@@ -21,6 +21,15 @@
             <span class="form-text text-muted">Napríklad: 36421928</span>
           </div>
         </div>
+        <div class="mt-3 row col-md-12">
+          <%= toggle_switch_tag :api_interest, params[:mode], 'mr-2' %>
+          <span>Mám záujem o službu
+            <a data-toggle="tooltip">
+              <em> GovBox API.</em>
+              <span style="right: 0">Umožňuje manipuláciu s schránkou prostredníctvom programového rozhrania (API) najmä prístup do elektronickej schránky, sťahovanie elektronických správ a elektronických dokumentov do vlastného rozhrania, potvrdzovanie doručeniek ako aj ďalšie funkcionality.</span>
```

Comment:
Tento tooltip nevyzera dobre. Ludia na to nebudu nabiehat myskou a zaroven na mobile to nemas ako precitat.  V klude to dajme ako poznamku.  Taktiez by som to presunul na iny krok registracie, nie uplny uvod. Logicky zmysel ma tento checkbox v kroku nastavenie sluzby (treba zmenit podnadpis). 

Taktiez by som asi vyrazil ten fancy checkbox lebo som za konzistentnost. Na inom kroku mame normalny checkbox, toto potom vyzera divne.

Reply:
Ja som ho spravil responsivne tak aby siel aj na mobile. Ale dam ho prec (tooltip)
Aj fancy checkbox

Reply:
![image](https://user-images.githubusercontent.com/44023416/65604602-0ac9ae00-dfa8-11e9-9af8-d233720e73dc.png)

Reply:
Ten toggle switch sa mi pacil viacej ale mas pravdu tooltip netreba